packages/@okta/vuepress-site/docs/guides/unlock-mobile-app-with-biometrics/discard-access-tokens/index.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Discard access tokens
+---
+By discarding the current access token, the mobile app is prevented from making any API requests. To get a new access token, the refresh token must be unlocked using a biometric factor. You can discard an access token after a period of time (as configured in your Okta settings). You can also revoke and discard access tokens when the app closes, if you want the user to be prompted for a biometric challenge every time that they re-open the app.
```

Comment:
In the Android SDK you can't specifically delete the access token only. It only have a method that removes all the tokens. I'm not sure how it is in iOS but can we change the wording for this? The developer should call revoke token if they want to target a specific token. We can add this functionality in the future.

Reply:
The same for iOS, if you call `delete`, library deletes all the data(access token, refresh token and etc.). So this chapter is a bit confusing

Probably this chapter should show how to delete tokens from the secure storage and related `biometric challenge`?

Reply:
@IldarAbdullin-okta and @FeiChen-okta 

This title could remain the same, right? "Discard access tokens", but show how to delete tokens from secure storage and the biometric challenge that is associated with that?

Reply:
Yes I think we can keep the title.